scripts/mutation-triage.sh

```
@@ -0,0 +1,106 @@
+
+  local backup log_file patch_file
+  backup="$(mktemp)"
+  log_file="$(mktemp)"
+  patch_file="$(mktemp)"
+
+  echo "=== $id ==="
+  if ! cp "$file" "$backup"; then
+    rm -f "$backup" "$log_file" "$patch_file"
+    return 1
+  fi
+  trap "restore_backup '${backup}' '${file}'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)